### PR TITLE
Sfr 1215 clear search terms

### PR DESCRIPTION
**Pull request by Toxiapo**
[Jira Ticket](http://jira.nypl.org/browse/SFR-1215)

### This PR does the following:
- Clicking `Advanced Search` should start a new search. 

### Testing requirements & instructions: 
-  Make sure all the search fields are cleared out on the advanced-search page each time we start a search.

### CODE REVIEW

File: src/__tests__/AdvancedSearch.test.tsx [hunk 1]
```diff
@@ -83,48 +52,46 @@ describe("renders advanced search correctly", () => {
...
+      Title: "Handbook",
+    };
+    inputTerms.forEach((val) => {
+      fireEvent.change(screen.getByLabelText(val.label), {
+        target: { value: inputValues[val.label] },
+      });
+    });
+
+    userEvent.click(screen.getByRole("checkbox", { name: "english" }));
+    fireEvent.change(screen.getByRole("spinbutton", { name: "From" }), {
```
Comment (hokei) [inline src/__tests__/AdvancedSearch.test.tsx:83]: Looks like we test for year, but not for all the other filters (format, language) - can we just make a change to all of them and make sure they all work?

File: src/__tests__/AdvancedSearch.test.tsx [hunk 2]
```diff
@@ -225,13 +193,21 @@ describe("Advanced Search clear", () => {
...
-        <AdvancedSearch
-          searchQuery={complicatedSearchQuery}
-          languages={defaultLanguages}
-        />
+        <AdvancedSearch languages={defaultLanguages} />
       </MockNextRouterContextProvider>
     );
 
+    userEvent.click(screen.getByRole("checkbox", { name: "english" }));
+    fireEvent.change(screen.getByRole("spinbutton", { name: "From" }), {
```
Comment (hokei) [inline src/__tests__/AdvancedSearch.test.tsx:201]: Same as above - fill in everything and make sure it all clears.

**Review by hokei — CHANGES_REQUESTED** [review]
Looks really good.  Some additional test changes, and it should be good to go.

File: src/__tests__/AdvancedSearch.test.tsx (hunk 1 above)
Comment (Toxiapo) [reply src/__tests__/AdvancedSearch.test.tsx:83]: Yep! I must've missed the format.

**Review by hokei — APPROVED** [review]
Looks good!  

At this point, you should paste the DRB front-end url that was posted in the bot, into the comment of the ticket, and move it to "Ready for QA" - the branch should be merged into development only after it's been signed off by QA.

**Pull request merged**